src/internal/methods/NeoArmMethod.h

```diff
@@ -204,10 +204,13 @@ template<typename T_SPEEDPROPS> class NeoArmMk20dxSpeedBase
         uint8_t pix;
         uint8_t mask;
 
-        volatile uint8_t* set = portSetRegister(pin);
-        volatile uint8_t* clr = portClearRegister(pin);
+        volatile auto set = portSetRegister(pin);
+        volatile auto clr = portClearRegister(pin);
 
         uint32_t cyc;
+        #if !defined(KINETIS)
```

Comment:
What symbol is this "KINETIS"?  You didn't add it above to get into this area, why is this different?
Also, just if def here and just use msk below
like
```
#if defined(KINETIS)
const uint32_t msk = digitalPinToBitMask(pin);
#else
const uint32_t msk = 1;
#endif
```

Reply:
Yes we can do that
Kinetis is the define present in any teensy 3.x and not 4.1 so this is used to differentiate

Reply:
Yes we can use like this. I only separated them because for KINETIS ones we are using uint8_t and for 4.0 and 4.1 we are using uint32_t, so I'm unsure. in this PR "auto" is replacing both, but we can use other resource as typedef.

Reply:
thinking better we can use like you said, but
```
#if defined(KINETIS)
const uint32_t msk = digitalPinToBitMask(pin);
#else
const uint8_t msk = 1;
#endif
```

Reply:
good. I followed your logic but only changed data types to fit differences between 4.0/4.1 and the rest